core/chains/evm/types/nonce.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import (
+	"fmt"
+	"strconv"
+)
+
+var zeroNonce = Nonce(0)
+var _ fmt.Stringer = &zeroNonce
```

Comment:
Does this compile with the pointer type? Given the value type receivers `(n Nonce)`, I would expect this:
```suggestion
var _ fmt.Stringer = zeroNonce
```

Reply:
Hmm, yes it was compiling with pointer type.
But now changed to 
var _ fmt.Stringer = Nonce(0)

Thanks!